src/System.Management.Automation/utils/CryptoUtils.cs

```diff
@@ -628,7 +628,7 @@ protected void Dispose(bool disposing)
         }
 
         /// <summary>
-        /// Destructor.
+        /// Finalizes an instance of the <see cref="PSRSACryptoServiceProvider"/> class.
         /// </summary>
         ~PSRSACryptoServiceProvider()
         {
```

Comment:
I wonder why we call `Dispose(true)` not `Dispose(false)` in the finalizer.

Reply:
Please open new PR for discussion.

Reply:
Submitted #14246.